main.js

```diff
@@ -214,8 +215,11 @@ $catalog.addEventListener('click', function (event) {
 
 $details.addEventListener('click', function (event) {
   var $clicked = event.target
-  if ($clicked.closest('.btn') !== null) {
+  if ($clicked.closest('.btn-primary') !== null) {
```

Comment:
It's better to use dedicated attributes like `id` or `data-whatever`. You don't want to couple your JavaScript to your styles unnecessarily.
https://searchnetworking.techtarget.com/definition/loose-coupling